src/posts/vector-data-cubes/index.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Vector data cubes in Xarray
+description: Thanks to Xvec and developments across a number of packages, the Xarray ecosystem now supports data cubes with vector geometries as coordinate locations.
```

Comment:
```suggestion
date: '2024-08-15'
authors:
  - name: Emma Marshall
    github: e-marshall
summary: 'Thanks to Xvec and developments across a number of packages, the Xarray ecosystem now supports data cubes with vector geometries as coordinate locations.'
```

Reply:
☝🏽 adds missing metadata that are necessary for the build process to pass.

Reply:
ah thanks @andersy005 ! didn't realize those were required